src/schema/core.yaml

```diff
@@ -137,13 +137,14 @@ classes:
       - name
     todos:
       - add additional fields e.g for institution
+      - deprecate "has raw value" in favor of "name"
     slot_usage:
       has raw value:
         description: >-
-          the full name of the Investgator in format FIRST LAST
+          The full name of the Investgator in format FIRST LAST. May eventually be deprecated in favor of "name".
```

Comment:
Information about depreciation is something that would go in a note. I would not include it in a description. Also, you may want to in information about an optional middle name or initial. E.g.:
```
description: >-
  The name of the person.   
  It should follow the format FIRST [MIDDLE NAME| MIDDLE INITIAL] LAST, where MIDDLE NAME| MIDDLE INITIAL is optional.